MAINTAIN.md

```diff
@@ -60,9 +60,36 @@ Adding new presets
 ------------------
 
 1. Make sure they follow the rules listed in CONTRIBUTING.md
-1. Discuss the preset on the mailing list: jscs-dev@googlegroups.com
+1. Discuss the preset on the [mailing list](jscs-dev@googlegroups.com) or [gitter room](https://gitter.im/jscs-dev/node-jscs)
 1. Once discussed and agreed upon, land the PR!
 
+Merging a PR
+------------
+
+We never hit the Merge pull request button on github. This avoids merge commits which makes history muddy.
+We generate changelogs from the history, so the less noise – the better.
+
+We never change history on master.
+If you make a mistake and push up a silly commit to upstream's master, then you issue a revert commit, you do not remove the commit from the history and force push up.
+
+1. Checkout the PR locally `hub checkout the_pull_request_url`
+2. Make sure that the PR is up to date with master `git pull --rebase upstream master`
```

Comment:
We should update the release instruction as well, since in there it assumes your copy points to origin